### PR TITLE
Make printf spec C-standard compliant and some minor bug fixes

### PR DESCRIPTION
**Pull request by selvanair**
Commits 1 and 2 are minor bug-fixes.
Commit 3:  Some wide strings are truncated to single character when built with latest mingw-w64 because of changes related to implicit enabling of __USE_MINGW_ANSI_STDIO with _GNU_SOURCE. 

Assigned IP for example: truncation

Instead of a quick-fix,  use standard "%ls" or "%hs" instead of %s and %S for wide strings everywhere. Majority of the changes (~970 lines) are in resource files made using sed. The rest (~80 lines) edited manually.

### CODE REVIEW

**Comment by lstipakov** [comment]
Do you want to squash https://github.com/OpenVPN/openvpn-gui/pull/459/commits/83997829dc5512c626938f0f08ea5ec2deba4f25 into https://github.com/OpenVPN/openvpn-gui/pull/459/commits/4041cd670425e5f1adb5969b17d544609b0ba19d ?

**Comment by selvanair** [comment]
> Do you want to squash [8399782](https://github.com/OpenVPN/openvpn-gui/commit/83997829dc5512c626938f0f08ea5ec2deba4f25) into [4041cd6](https://github.com/OpenVPN/openvpn-gui/commit/4041cd670425e5f1adb5969b17d544609b0ba19d) ?

Yes, its generated using `git --fixup` with intention to do `git --autosquash` before merge. I can do that now and re-push, if preferred.

**Comment by lstipakov** [comment]
Looks like sed only replaced first occurrence in string?

**Comment by selvanair** [comment]
> Looks like sed only replaced first occurrence in string?

oops, I missed /g in the sed expression. Will check again and push.

**Comment by selvanair** [comment]
v2: multiple occurrences on same line was missed in v1 (/g missing in sed expression) -- res/*.rc files only. Fixed, squashed, and force-pushed.

**Review by lstipakov — APPROVED** [review]

File: res/openvpn-gui-res-en.rc [hunk 1]
```diff
@@ -512,32 +512,32 @@ OpenVPN is probably not installed"
...
+    IDS_NFO_TOKEN_PASSWORD_REQUEST "Input Password/PIN for Token '%hs'"
 
     IDS_NFO_AUTH_PASS_RETRY "Wrong credentials. Try again…"
     IDS_NFO_KEY_PASS_RETRY  "Wrong password. Try again…"
     IDS_ERR_INVALID_PASSWORD_INPUT "Invalid character in password"
     IDS_ERR_INVALID_USERNAME_INPUT "Invalid character in username"
     IDS_NFO_AUTO_CONNECT    "Connecting automatically in %u seconds…"
     IDS_NFO_CLICK_HERE_TO_START "OpenVPN GUI is already running. Right click on the tray icon to start."
-    IDS_NFO_BYTECOUNT "Bytes in: %s  out: %s"
+    IDS_NFO_BYTECOUNT "Bytes in: %ls  out: %s"
```
[the remark this answers is not in the document]
Comment (selvanair) [reply res/openvpn-gui-res-en.rc]: I fixed these and pushed a squashed version yesterday (the sed missing /g). Is it still showing the old commits or is github out of sync? Same with your comments below. 

Not sure why the updated files are not showing up for you.... assuming that's the case.

[the remark this answers is not in the document]
Comment (lstipakov) [reply res/openvpn-gui-res-en.rc]: Those are outdated comments, as GitHub mentioned. I approved this PR.

**Pull request merged**